app/js/sumo_db.js

```diff
@@ -161,6 +161,15 @@
       return request(endpoint, 'GET').then(JSON.parse);
     },
 
+    authenticate_user: function(username, password) {
+      var endpoint = API_V1_BASE + 'users/get_token';
+      var data = {
+        username: username,
+        password: password
+      };
+      return request(endpoint, 'POST', data).then(JSON.parse);
```

Comment:
That looks very similar to the already existing get_token method (but not used at the moment). I'd prefer if we keep the get_token name because it will also be useful when our token expires.